webui/react/src/hooks/usePermissions.ts

```diff
@@ -623,7 +623,8 @@ const canManageResourcePoolBindings = ({
 }: RbacOptsProps): boolean => {
   const permitted = relevantPermissions(userAssignments, userRoles);
   return rbacEnabled
-    ? permitted.has(V1PermissionType.UPDATEMASTERCONFIG)
+    ? permitted.has(V1PermissionType.UPDATEMASTERCONFIG) ||
+        permitted.has(V1PermissionType.UPDATEWORKSPACE)
```

Comment:
I don't see one that is specific to RP bindings for `V1PermissionType` but let me know if this is acceptable.

Reply:
There is `rbacv1.PermissionType_PERMISSION_TYPE_MODIFY_RP_WORKSPACE_BINDINGS` in the backend. @eecsliu Required the same in web?

Reply:
oh ok I see it (`MODIFYRPWORKSPACEBINDINGS`), I'll make the changes

Reply:
RESOLVED: discussed async, having Workspace Update capability is the behavior we want